share/packages/orca_functional_names.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<functionals>
+  <XC_HYB_GGA_XC_B1LYP number="416">B1LYP</XC_HYB_GGA_XC_B1LYP>
+  <XC_HYB_GGA_XC_B3LYP number="402">B3LYP</XC_HYB_GGA_XC_B3LYP>
+  <XC_HYB_GGA_XC_PBEH number="406">PBE0</XC_HYB_GGA_XC_PBEH>
+  <XC_GGA_C_PBE number="130">PBE</XC_GGA_C_PBE>
```

Comment:
this line is problemati because the PBE functional is XC_GGA_C_PBE + XC_GGA_X_PBE,

C and X indicate the correlation and exchange part separately.

Reply:
mmmm, so when you are using PBE do you call functional as:
 ```
XC_GGA_C_PBE+XC_GGA_X_PBE
```
Or how do you call it?

Reply:
XC_GGA_C_PBE XC_GGA_X_PBE

Reply:
the order does not matter